var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
```

Comment:
Done.
But I'm a bit confused atm, what is the desired default for a new package: Having a general ```url=...``` or one for each version?